impl/src/rendering/bazel.rs

```diff
@@ -351,16 +389,10 @@ impl BuildRenderer for BazelRenderer {
 
     // Ensure there is always a `BUILD.bazel` file to accompany `crates.bzl`
     let crates_bzl_pkg_file = path_prefix.as_path().join("BUILD.bazel");
```

Comment:
Move this into the render_crates_bzl_package_file function?

Reply:
Done